charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```
@@ -0,0 +1,74 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    role: {{ .Values.role }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podAnnotations }}
+  annotations:
+{{ toYaml .Values.podAnnotations | indent 4 }}
+{{- end }}
+  selector:
+    matchLabels:
+      garden.sapcloud.io/role: controlplane
+      app: etcd-statefulset
+      role: {{ .Values.role }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
+    networking.gardener.cloud/to-dns: allowed
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/to-private-networks: allowed
+  etcd:
+    tls:
+      serverTLSSecretRef:
+        name: {{ .Values.tlsServerSecretName }}
+      clientTLSSecretRef:
+        name: {{ .Values.tlsClientSecretName }}
+      tlsCASecretRef:
+        name: {{ .Values.tlsCASecretName }}
+    metrics: {{ .Values.metrics }}
+    image: {{ index .Values.images "etcd" }}
+    defragmentationSchedule: {{ .Values.defragmentSchedule }}
+{{- if .Values.etcdResources }}
+    resources:
+{{ toYaml .Values.etcdResources | indent 6 }}
+{{- end }}
+    clientPort: {{ .Values.servicePorts.client }}
+    serverPort: {{ .Values.servicePorts.server }}
+  backup:
+    image: {{ index .Values.images "etcd-backup-restore" }}
+    port: 8080
+{{- if .Values.sidecarResources }}
+    resources:
+{{ toYaml .Values.sidecarResources | indent 6 }}
+{{- end }}
+    garbageCollectionPolicy: Exponential
+    quota: 8Gi
+    garbageCollectionPeriod: 12h
```

Shouldn't this be configurable from values.yaml?
```suggestion
    garbageCollectionPeriod: {{ .Values.backup.garbageCollectionPeriod }}
```

---

Makes sense.